### PR TITLE
fix pytest warnings in 3.7

### PR DESCRIPTION
**Pull request by ChristopherChudzicki**

### CODE REVIEW

File: tests/formulagrader/test_matrixgrader.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from __future__ import print_function, division, absolute_import
 
-from pytest import raises
 import re
+from pytest import raises
```
Comment (jolyonb) [inline tests/formulagrader/test_matrixgrader.py]: Really? That order mattered???

**Comment by jolyonb** [comment]
Looks fine to me. Down to 29 warnings! Do you want me to merge this, or are you going to add to it?

**Comment by ChristopherChudzicki** [comment]
Still working. For some reason not all of the warnings are showing in my local version

File: tests/formulagrader/test_matrixgrader.py (hunk 1 above)
Comment (ChristopherChudzicki) [reply tests/formulagrader/test_matrixgrader.py]: No, it didn't matter. But pylint likes it when standard-library imports come before other imports, and it was easy to change.

(It's the lowest category of pylint message.)

**Comment by ChristopherChudzicki** [comment]
Travis shows no warnings now. Ready for merge

**Comment by jolyonb** [comment]
Good  job!

**Pull request merged**